src/index.js

```diff
@@ -17,25 +22,67 @@ export default class Differencify {
   constructor(conf) {
     this.configuration = sanitiseGlobalConfiguration(conf);
     this.ChromyRunner = new ChromyRunner(this.configuration);
+    this.chromeInstances = {};
+    this.chromeInstancesId = CHROME_PORT;
     if (this.configuration.debug === true) {
       logger.enable();
     }
     createDir(this.configuration.screenshots);
     createDir(this.configuration.testReportPath);
   }
+  _createChromeInstance(testConfig) {
+    const width = testConfig.resolution.width || CHROME_WIDTH;
+    const height = testConfig.resolution.height || CHROME_HEIGHT;
+    const flags = [`--window-size=${width},${height}`];
+    const chromy = new Chromy({
+      chromeFlags: flags,
+      port: this.chromeInstancesId,
+      waitTimeout: this.configuration.timeout,
+      visible: this.configuration.visible || false,
+    });
+    return chromy;
+  }
+  _updateChromeInstances(id, chromy) {
+    this.chromeInstances[id] = chromy;
+    this.chromeInstancesId += 1;
+  }
+  async _closeChrome(id, chromy) {
+    try {
+      logger.log('closing browser');
+      chromy.close();
+      delete this.chromeInstances[id];
+    } catch (error) {
+      logger.error(error);
+    }
+  }
   async update(config) {
     const testConfig = sanitiseTestConfiguration(config);
```

Comment:
There's now a tonne of overlapping functionality between `update` and `run`, only two lines differ. It's probably worth refactoring